front/src/modules/simulationResult/styles/simulationResult.scss

```
@@ -8,6 +8,25 @@
   transition: 0.2s;
 }
 
+.right-column {
+  height: 100%;
+  position: relative;
+}
+
+.conflicts-wrapper {
+  display: block;
+  position: relative;
+  width: 100%;
+  transition: 0.2s;
+  max-height: 90vh;
+  overflow-y: auto;
+
+  .scenario-loader {
+    position: relative;
+    height: 100%;
+  }
+}
```

Here the `scenario-loader` should be `position: absolute` so it’s positioned relative to `.conflicts-wrapper` (which is` position: relative`).

The loader didn’t take the full height because height: 100% on a relative element only works if the parent has an explicit height; when the list is empty, the wrapper collapses.

With the change below, the loader is anchored to the wrapper and stretches to fill it:
```suggestion
.conflicts-wrapper {
  display: flex;
  flex-direction: column;
  flex: 1;
  position: relative;
  transition: 0.2s;
  overflow: hidden auto;

  .scenario-loader {
    position: absolute;
  }
}
```